qiskit_ibm_provider/ibm_backend.py

```diff
@@ -505,52 +497,90 @@ def run(
             init_qubits=init_qubits,
             use_measure_esp=use_measure_esp,
             noise_model=noise_model,
+            max_circuits_per_job=max_circuits_per_job,
+            parameter_binds=parameter_binds,
+            live_data_enabled=live_data_enabled,
             **run_config,
         )
-        if parameter_binds:
-            run_config_dict["parameter_binds"] = parameter_binds
-        if sim_method and "method" not in run_config_dict:
-            run_config_dict["method"] = sim_method
-
-        if isinstance(circuits, list):
-            chunk_size = None
-            if hasattr(self.configuration(), "max_experiments"):
-                backend_max = self.configuration().max_experiments
-                chunk_size = (
-                    backend_max
-                    if max_circuits_per_job is None
-                    else min(backend_max, max_circuits_per_job)
-                )
-            elif max_circuits_per_job:
-                chunk_size = max_circuits_per_job
-
-            if chunk_size and len(circuits) > chunk_size:
-                circuits_list = [
-                    circuits[x : x + chunk_size]
-                    for x in range(0, len(circuits), chunk_size)
-                ]
-                return IBMCompositeJob(
-                    backend=self,
-                    api_client=self._api_client,
-                    circuits_list=circuits_list,
-                    run_config=run_config_dict,
-                    name=job_name,
-                    tags=job_tags,
-                )
 
-        qobj = assemble(circuits, self, **run_config_dict)
+        return self._runtime_run(
+            program_id=program_id,
+            inputs=run_config_dict,
+            options=options,
+            job_tags=job_tags,
+        )
 
-        return self._submit_job(qobj, job_name, job_tags, live_data_enabled)
+    def _is_qasm3_string(
+        self,
+        circuit: Union[
+            str, QuantumCircuit, Schedule, List[Union[QuantumCircuit, Schedule]]
+        ],
+    ) -> bool:
+        """Checks if circuit is a a qasm3 string."""
+        if not isinstance(circuit, str):
+            return False
+        if re.search("OPENQASM 3", circuit) is None:
+            return False
+        return True
+
+    def _runtime_run(
+        self,
+        program_id: str,
+        inputs: Dict,
+        options: Dict,
+        job_tags: Optional[List[str]] = None,
+    ) -> IBMCircuitJob:
+        """Runs the runtime program and returns the corresponding job object"""
+        hgp = self.provider._get_hgps()[0]
+        hgp_name = hgp.name
+        try:
+            response = self.provider._runtime_client.program_run(
+                program_id=program_id,
+                backend_name=options["backend"],
+                params=inputs,
+                image=options.get("image"),
```

Comment:
I don't think `image` and `log_level` are ever set in `options` passed to this method?

Reply:
You're right, these have been be removed